src/cmd/build.js

```diff
@@ -12,9 +12,47 @@ import {createLogger} from '../util/logger';
 
 const log = createLogger(__filename);
 
+// Flow Types
+
+import type {OnSourceChangeFn} from '../watcher';
+import type {ExtensionManifest} from '../util/manifest';
+
+export type PackageCreatorFn =
+    (params: PackageCreatorParams) => Promise<ExtensionBuildResult>;
+
+export type ExtensionBuildResult = {
+  extensionPath: string,
+};
+
+export type PackageCreatorParams = {
```

Comment:
yep, I forgot to move it :-(  
I'm going to push a fix after this round of review comment.